programs/programs/wa/lifeline/tests/test_wa_lifeline.py

```
@@ -0,0 +1,137 @@
+
+
+class TestWaLifelineClassAttributes(TestCase):
+    def test_is_registered(self):
+        self.assertIn("wa_lifeline", wa_calculators)
+        self.assertEqual(wa_calculators["wa_lifeline"], WaLifeline)
+
+    def test_fpl_percent(self):
+        self.assertEqual(WaLifeline.fpl_percent, 1.35)
+
+    def test_amount(self):
+        # $9.25/month × 12 = $111/year
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)